engine/engine.go

```diff
@@ -334,7 +334,7 @@ func (r *ruleEngine) runTaggingRules(ctx context.Context, infoRules []ruleMessag
 				rs.Errors[rule.RuleID] = err.Error()
 			}
 		} else if response.Matched && len(response.Incidents) > 0 {
-			r.logger.V(5).Info("info rule was matched", "ruleID", rule.RuleID)
+			r.logger.V(5).Info("rule was matched", "ruleID", rule.RuleID)
```

Comment:
why remove info here?

Reply:
@shawn-hurley it sounded like a rule of type "info" was matched. It was confusing to me, as in windup we used to have INFO rules. But I can put it back if you want

Reply:
can we updated to tagging rules, becuase they are seperate from the non-tagging rules

Reply:
Then it probably makes sense to leave it as it was